scripts/check-prometheusrules.py

```
@@ -0,0 +1,411 @@
+    _check_runbook_url,
+    _check_namespace,
+]
+
+
+def _run_promtool(path, doc):
+    spec = doc.get('spec', {})
+    with tempfile.NamedTemporaryFile(mode='w', suffix='.json', delete=False) as f:
+        json.dump(spec, f)
+        tmp_path = f.name
+    try:
+        result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)